test/event_queue_test.c

```diff
@@ -31,38 +31,46 @@ static int event_test(void)
 
     /* Create an event queue and add some events */
     if (!TEST_ptr(q = ossl_event_queue_new())
-            || !TEST_ptr(e1 = ossl_event_queue_add_new(q, 1, 10, 1100, "ctx 1",
+            || !TEST_ptr(e1 = ossl_event_queue_add_new(q, 1, 10,
+                                                       ossl_ticks2time(1100),
+                                                       "ctx 1",
                                                        PAYLOAD(payload)))
-            || !TEST_ptr(e2 = ossl_event_queue_add_new(q, 2, 5, 1100, "ctx 2",
+            || !TEST_ptr(e2 = ossl_event_queue_add_new(q, 2, 5,
+                                                       ossl_ticks2time(1100),
+                                                       "ctx 2",
                                                        PAYLOAD("data")))
-            || !TEST_true(ossl_event_queue_add(q, &e3, 3, 20, 1200, "ctx 3",
+            || !TEST_true(ossl_event_queue_add(q, &e3, 3, 20,
+                                               ossl_ticks2time(1200), "ctx 3",
                                                PAYLOAD("more data")))
-            || !TEST_ptr(e4 = ossl_event_queue_add_new(q, 2, 5, 1150, "ctx 2",
+            || !TEST_ptr(e4 = ossl_event_queue_add_new(q, 2, 5,
+                                                       ossl_ticks2time(1150),
+                                                       "ctx 2",
                                                        PAYLOAD("data")))
 
             /* Verify some event details */
             || !TEST_uint_eq(ossl_event_get_type(e1), 1)
             || !TEST_uint_eq(ossl_event_get_priority(e1), 10)
-            || !TEST_uint64_t_eq(ossl_event_get_when(e1), 1100)
+            || !TEST_uint64_t_eq(ossl_event_get_when(e1).t, 1100)
```

Comment:
Shouldn't this be using `ossl_time2ticks`? Same for other similar instances below.